src/olympia/migrations/1029-delete-localization-groups.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM groups_users WHERE group_id IN (SELECT id FROM groups WHERE name LIKE "% Localizers");
+DELETE FROM groups WHERE name LIKE "% Localizers";
```

Comment:
I was a bit confused about the whitespace so for reference, this is how the groups were referenced:

``.filter(groupuser__group__name='%s Localizers' % lang)`` so this is fine.

Reply:
Verified both queries on sql.telemetry.mozilla.org, looks perfect.